app/javascript/redux/greetings/greetings.js

```diff
@@ -0,0 +1,31 @@
+const LOAD_GREETING = 'App/greetings/LOAD_GREETING';
+
+const URL = 'http://localhost:3000/v1/greetings';
+
+const initialState = {
+    id: 1000,
+    greeting: "hey mr robot"
+};
+
+export const loadGreeting = (payload) => ({
+    type: LOAD_GREETING,
+    payload
+});
+
+export const getGreetings = () => async (dispatch) => {
+    await fetch(URL)
+    .then((res) => res.json())
+    .then((greeting) => dispatch(loadGreeting(greeting)))
+    .catch((err) => console.log(err))
+}
```

Comment:
- [ ] Greeting is not randomly generated. . There are errors found in the console which are likely traceable to the above error. Kindly improve upon this aspect by ensuring that random greetings are generated when a button is clicked or when the page is refreshed.

**Errors:**
![image](https://user-images.githubusercontent.com/12745474/150122993-7749c166-0209-4527-810b-865141c792ba.png)

Reply:
I checked and re-checked I have no errors in my part and all things working well as described in the requirements as you can see in the screenshot below 
![Kazam_screenshot_00007](https://user-images.githubusercontent.com/80895497/150185718-baf87dc7-eadc-4c00-b6b6-4adb68311427.png)
 - Did you run `rails db:seed` before starting the server ?

Reply:
@Hamzaoutdoors 
Absolutely.. The seed command was run